### PR TITLE
Split aggregates_per_concept to two: aggregates_per_numerical_concept and aggregates_per_categorical_concept

### PR DESCRIPTION
**Pull request by forus**

### CODE REVIEW

**Comment by ewelinagr** [comment]
There is an e2e test class `AggregatesPerConceptSpec`, is it still useful to have tests for deprecated call? Maybe better to remove it or change to use the new calls?

File: open-api/swagger.yaml [hunk 1]
```diff
@@ -3125,32 +3174,56 @@ definitions:
     type: object
     properties:
       numericalValueAggregatesPerConcept:
+        $ref: '#/definitions/NumericalAggregates'
+      categoricalValueAggregatesPerConcept:
+        $ref: '#/definitions/NumericalAggregates'
```
Comment (ewelinagr) [inline open-api/swagger.yaml]: seems like it should be ` $ref: '#/definitions/CategoricalAggregates'` instead

Comment (forus) [reply open-api/swagger.yaml]: nice catch. thanks!

**Comment by forus** [comment]
@ewelinagr we will remove e2e tests when we delete the call.
I thought about e2e tests for the new calls. But the functional tests in the `transmart-rest-api` seem to cover much logic already. I am not sure what e2e would add up.

**Pull request merged**